scripts/test_pr_build.sh

```diff
@@ -5,6 +5,20 @@
 # SPACK_PRODUCTION_DIR: path where the production instance of Spack resides
 #
 
+# Configure ccache properly. Caches are kept separated among different targets.
+mkdir -p ~/.ccache/${SPACK_TARGET_TYPE}
+mkdir -p /scratch/scitasbuild/ccache/${SPACK_TARGET_TYPE}
+
+export CCACHE_CONFIGPATH=~/.ccache/${SPACK_TARGET_TYPE}/ccache.conf
```

Comment:
Could you add a couple of variables more here and avoid repeating path names, for example: "CCACHE_CONFIG_DIR" and "CCACHE_CACHE_DIR". (Not exported.)